setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
I'd go for ```scitools-iris``` as per [PEP0423](http://legacy.python.org/dev/peps/pep-0423/#top-level-namespace-relates-to-code-ownership) (note: PEP status=deferred, but it does give some compelling evidence)